### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by Mightychocolate** [sircharlo/meeting-media-manager#477]
**Describe the bug**
The audio coming across to zoom is very muffled and the camera cokes across blurry on zoom. We are using OBS 

**Expected behavior**
Just wanted clear audio and clarity on zoom camera. 

**System specs (please complete the following information):**
 - Windows 10
 

**Additional context**
Is there anything we can do to improve?

**Comment by mtdvlpr** [comment]
Hi @Mightychocolate, have you checked out [the documentation](https://mtdvlpr.github.io/meeting-media-manager/en/#present-media) yet? It has a heading "Initial configuration: OBS Studio" which lists some preferences to set in OBS. For the Zoom part, you could look into [Zoom's original sound option](https://support.zoom.us/hc/en-us/articles/115003279466-Using-original-sound-for-musicians-in-a-meeting#:~:text=Zoom%20desktop%20client.-,Start%20or%20join%20a%20meeting.,toggle%20this%20on%20or%20off.). Hope this helps!

**Comment by Mightychocolate** [comment]
Yeah I have set up everything to the T, still to no avail.

But haven't tried just using zooms audio would that be better?

**Comment by Chipperchap** [comment]
I do not recommend using the "Original Sound" option - at least it was a disaster for us. 

What has worked perfectly for us is the "Share - Advanced - Computer Audio". The sound is crystal clear for the participants, both from the PC (M3 videos) and the meeting participants / mixer audio. I have set up a "Task Scheduler" to provide a message prompt for each meeting to use the "Share audio" as there is no known way to get it to share automatically. We have had no problems since using this option as also are the other congregations I assist. It does mean that for every ZOOM meeting there is a banner at the top that reads "the host is sharing computer audio" but no one has commented on it and the alternative is simply out of the question.  If there was some way to not have to do it would be great, but we simply do not have the audio infrastructure at our K-Hall. 

FYI we only have an old mixer/amp so use the "REC OUT" option connected to a Behringer UCA222 RCA to USB interface unit which connects to the PC for sound. from the microphones.  (This is the LDC recommended unit in this part of the world). 

As far as video quality is concerned - we use a LDC specs PTZ camera which provides a wonderful picture to the host PC and ZOOM (via OBS), but for some reason what is sent out from ZOOM to the recipients / participants is less than perfect, almost fuzzy, but I have never been able to put my finger on why. It might be something to do with the onboard graphics card... or maybe ZOOM just downgrades the image to make it easier to send out when the internet bandwidth is not great. In other words it has nothing to do with, as far as I can ascertain, OBS.

**Comment by Mightychocolate** [comment]
So to confirm you are not using the sound coming from OBS at all? And just sharing computer sound?

**Comment by Chipperchap** [comment]
I have spent countless hours with OBS and can say that I am reasonably familiar with it, however despite those hours and multiple trials and experiments I have failed to get the OBS audio to work for us. So, in short, no, I avoid the OBS audio feature. We only use OBS for the virtual camera, PTZ presets and control, and its integration with M<sup>3</sup> media for ZOOM which has been absolutely amazing. 🥰

On the other hand, if anyone has been able to get OBS audio to work for them so that they don't have to use the ZOOM "Share Computer Audio" feature then I am all ears. When we started using ZOOM I tried 'virtual cables' as well but they make my head hurt trying to figure out how to use them effectively. 

I will probably be shown how and think "of course! Why didn't I think of that!" 😁
